mava/components/jax/building/datasets.py

```diff
@@ -26,6 +27,38 @@
 Transform = Callable[[reverb.ReplaySample], reverb.ReplaySample]
 
 
+class TrainerDataset(Component):
```

Comment:
Very minor, but isn't it a bit more intuitive to call this `Dataset `instead?

Reply:
Agreed, changed

Reply:
Actually, switched it back to `TrainerDataset` to match the hook name. Sorry, that's why I named it that originally

Reply:
I see, that is fine by me in that case.